build/docker/build-tools.arm.dockerfile

```diff
@@ -0,0 +1,14 @@
+# As a basic image for building various components of KubeEdge
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y wget apt-utils \
```

Comment:
Correct me if I am wrong. Do we need this logic like if-else to combine these Dockerfiles? I think one Dockerfile would make sense.

Reply:
I'm ot sure whether the dockerfile can aware of the cpu arch or pass any parameter when building?

Reply:
> pass any parameter when building

Do we implement `ARG arch=<control parameter>` above the `FROM`. Then we can use this parameter to determine what the cmdline to execute.

Reply:
yes, like #3629 `edgecore.dockerfile`, we can use `ARG` above the `FROM`

Reply:
to specify the base image architecture for the build

Reply:
Updated, ptal :)